influxdb/rootfs/Dockerfile

```diff
@@ -1,8 +1,35 @@
-#This dockerfile is based on https://github.com/jalateras/docker-influxdb
-FROM alpine:3.3
-MAINTAINER Jonathan Chauncey "<jchauncey@deis.com>"
+FROM quay.io/deis/base:0.2.0
+
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /home/influxdb \
+	--group \
+	influxdb
+
+COPY . /
+
+RUN curl -L -o /home/influxdb/envtpl https://github.com/arschles/envtpl/releases/download/0.2.3/envtpl_linux_amd64 \
+	&& chmod +x /home/influxdb/envtpl \
+	&& apt-get update \
+	&& apt-get install -y apt-transport-https \
+	&& curl -sL https://repos.influxdata.com/influxdb.key | apt-key add - \
+	&& . /etc/lsb-release \
+	&& echo "deb https://repos.influxdata.com/${DISTRIB_ID,,} ${DISTRIB_CODENAME} stable" | tee /etc/apt/sources.list.d/influxdb.list \
+	&& apt-get update \
+	&& apt-get install -y influxdb \
```

Comment:
we can do the latest which is fine but i dont want to have this thing potentially install a different version of influx just because we rebuild the image